### PR TITLE
8 15pt add a food list

### LINKED ISSUE #8: [1.5pt] Add a Food list
- Should display a list of food added by the logged-in user as in the wireframe
- Should display the quantity of a given food
- Should lead to a form that allows users to add new food

### PR DESCRIPTION
**Pull request by Trictonicmp**
#### In this PR I:

- [x] Added foods route
- [x] Added foods controller
- [x] Added foods list view

### CODE REVIEW

File: app/views/foods/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<h2>
+  Here's a list of your foods!
+</h2>
+<%= link_to(foods_path) do %>
```
Comment (JuanLPalacios) [inline app/views/foods/index.html.erb]: - [x] [REQUIRED] foods path is the current path it should leed to where new food is added. use  `new_food_path` instead

File: app/views/foods/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+<% end %>
+<ul>
+  <% @foods.each do |food| %>
+    <li>
+      <span>Food</span>
+      <span>Measurement unit</span>
+      <span>Unit price</span>
+      <span>Quantity</span>
+      <span>Actions</span>
+    </li>
```
Comment (JuanLPalacios) [inline app/views/foods/index.html.erb]: - [x] [OPTIONAL]These are the table titles row and should be out of the cycle each

**Review by JuanLPalacios — CHANGES_REQUESTED** [review]
Hi, @Trictonicmp.

Good job so far!
There are some issues that you still need to work on, but you are almost there!

### Highlights:
No linter issues 👍
Clean structured code

### Required Changes: ♻️

- add food link should lead to a form that allows users to add new food (check comments)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

File: app/views/foods/index.html.erb (hunk 2 above)
Comment (Trictonicmp) [reply app/views/foods/index.html.erb]: Oh man hahaha I didn't see I put it inside the each method, thank!

**Review by JuanLPalacios — APPROVED** [review]
Hi, @Trictonicmp .

STATUS: APPROVED! 🟢
Well done

Your project is complete! There is nothing else to say other than... it's time to merge it
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**